Makefile

```diff
@@ -36,8 +36,10 @@ NETWORK_CIDR := $(or $(NETWORK_CIDR),"192.168.126.0/24")
 NETWORK_NAME := $(or $(NETWORK_NAME), test-infra-net)
 NETWORK_BRIDGE := $(or $(NETWORK_BRIDGE), tt0)
 NETWORK_MTU := $(or $(NETWORK_MTU), 1500)
-PROXY_URL := $(or $(PROXY_URL), "")
 RUN_WITH_VIPS := $(or $(RUN_WITH_VIPS), "yes")
+HTTP_PROXY_URL := $(or $(HTTP_PROXY_URL), "")
```

Comment:
there is a possibility to have those params as OC_PARAMS env?
Not critical just can be nice

Reply:
@tsorya by looking at the purpose of OC_PARAMS - they seems to designed for the `oc` tool which isn't the case for HTTP_PROXY_URL. HTTP_PROXY_URL will be used to configure how agents should access the service and how to define the Proxy on the installed cluster. I don't OC_PARAMS is a good fit for them.